examples/health-status/README.md

```diff
@@ -0,0 +1,39 @@
+## Running Health Status Example
+
+This is a simple example, which does:
+
+1. Declare an "input-name" variable of type String
+2. Pass that variable into the execution of the "greet" task.
+3. Shows the cluster's health status.
+
+Let's run the example in `HealthStatus.java`
+
+```
+./gradlew example-health-status:run
+```
+
+In another terminal, use `lhctl` to run the workflow:
```

Comment:
How does this have something to do with the health status? The example isn't required (documentation _is_ required once we have added it to our OSS repo), but if you have it, please explain in the README how the health status thing works, and what the example does. Thanks!